index.js

```diff
@@ -32,6 +32,7 @@ module.exports = {
     'import/export': 'warn',
     'jest/no-focused-tests': 'error',
     'no-console': 'warn',
+    'no-unused-vars': 'error',
```

Comment:
Hey, can you explain your thinking behind having it as an error?
Overall I have a preference against it to be an error, for the following reasons:

* an unused variable isn’t a huge deal as to make it red. It doesn’t break anything, it is treeshaken away. Those small things are what warnings are for. The risk is that if we mark everything as important, then nothing is important anymore. 🤔 
* the coding experience gets worse… every time you start a new file and do “import React from ‘react’“, the linter screams at you in red because it is unused… and you are like”: “I know Linter, leave me in peace”. Then again as you type the next line, etc. While a yellow warning (like it is now) is much less agressive and irritating

Reply:
I understand what you mean, what I'm not sure is if there's a way to avoid a commit to go through with it being 'warn'. I've seen scenarios someone may miss this while going through files before pushing and they make sneak in.

Reply:
I suggest to run eslint with the option `--max-warnings` to `0`. Thus, it'll treat warnings as errors, but only for projects specifying it.

Reply:
Sounds good. Should we add it as 'warn' then? Or do you prefer not adding it at all?